project-base/storefront/components/Pages/OrderConfirmation/RegistrationAfterOrder.tsx

```diff
@@ -115,44 +114,47 @@ export const RegistrationAfterOrder: FC = () => {
 
             <div className="flex w-full flex-col items-center justify-center p-5 lg:w-1/2 lg:px-10 lg:py-8">
                 <div className="w-full lg:max-w-sm">
-                    <Form onSubmit={formProviderMethods.handleSubmit(onRegistrationHandler)}>
-                        <FormProvider {...formProviderMethods}>
-                            <PasswordInputControlled
-                                control={formProviderMethods.control}
-                                formName={formMeta.formName}
-                                name={formMeta.fields.password.name}
-                                passwordInputProps={{
-                                    label: formMeta.fields.password.label,
-                                }}
-                                render={(passwordInput) => (
-                                    <div className="mb-7">
-                                        <FormLine>{passwordInput}</FormLine>
-                                    </div>
-                                )}
-                            />
-                            <CheckboxControlled
-                                control={formProviderMethods.control}
-                                formName={formMeta.formName}
-                                name={formMeta.fields.privacyPolicy.name}
-                                checkboxProps={{
-                                    label: formMeta.fields.privacyPolicy.label,
-                                }}
-                                render={(checkbox) => (
-                                    <div className="mb-7">
-                                        <ChoiceFormLine>{checkbox}</ChoiceFormLine>
-                                    </div>
-                                )}
-                            />
+                    <FormProvider {...formProviderMethods}>
+                        <Form
```

Comment:
Why did we swap the hierarchy here?

Reply:
There was a bad hierarchy compared to the other places on the project, so I unified it